src/templates/partials/awselb.hbs

```diff
@@ -1,86 +1,51 @@
-                        ],
-                        "SSLCertificateId": {
-                            "Ref": "SSLCertificateId"
-                        },
-                        "Protocol": "HTTPS"
-                    }
-                ],
-                "AvailabilityZones": {
-                    "Fn::GetAZs": ""
-                },
-                "Policies": [
-                    {
-                        "PolicyName": "Mozilla-{{form.config}}-2015-03",
-                        "PolicyType": "SSLNegotiationPolicyType",
-                        "Attributes": [
+AWSTemplateFormatVersion: 2010-09-09
+Description: Mozilla ELB configuration generated {{output.date}}, {{{output.link}}}
+Parameters:
+  SSLCertificateId:
+    Description: The ARN of the ACM SSL certificate to use
+    Type: String
+    AllowedPattern: ^arn:aws:acm:[^:]*:[^:]*:certificate/.*$
+    ConstraintDescription: >
+      SSL Certificate ID must be a valid ACM ARN.
+      https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html#genref-arns
+Resources:
+  ExampleELB:
+    Type: AWS::ElasticLoadBalancing::LoadBalancer
+    Properties:
+      Listeners:
+        - LoadBalancerPort: '443'
+          InstancePort: '80'
+          PolicyNames:
+            - Mozilla-{{form.config}}-v5-0
+          SSLCertificateId: !Ref SSLCertificateId
+          Protocol: HTTPS
+      AvailabilityZones:
+        Fn::GetAZs: !Ref 'AWS::Region'
+      Policies:
+        - PolicyName: Mozilla-{{form.config}}-v5-0
+          PolicyType: SSLNegotiationPolicyType
+          Attributes:
 {{#if (includes "SSLv3" output.protocols)}}
-                            {
-                                "Name": "Protocol-SSLv3",
-                                "Value": true
-                            },
+            - Name: Protocol-SSLv3
+              Value: true
 {{/if}}
 {{#if (includes "TLSv1" output.protocols)}}
-                            {
-                                "Name": "Protocol-TLSv1",
-                                "Value": true
-                            },
+            - Name: Protocol-TLSv1
+              Value: true
 {{/if}}
 {{#if (includes "TLSv1.1" output.protocols)}}
-                            {
-                                "Name": "Protocol-TLSv1.1",
-                                "Value": true
-                            },
+            - Name: Protocol-TLSv1.1
+              Value: true
 {{/if}}
-                            {
-                                "Name": "Protocol-TLSv1.2",
-                                "Value": true
-                            },
-                            {
-                                "Name": "Server-Defined-Cipher-Order",
-                                "Value": true
-                            },
+            - Name: Protocol-TLSv1.2
+              Value: true
+            - Name: Server-Defined-Cipher-Order
+              Value: true
```

Comment:
This actually needs to get fixed to no longer be true, whoops.

Reply:
To clarify:

`{{#if output.serverPreferredOrder}}true{{else}}false{{/if}}`

Reply:
I've made this change